examples/inverse/plot_morph_stc_to_fsaverage_dipy.py

```diff
@@ -0,0 +1,231 @@
+"""
+==================================================================
+Demonstrate morphing subject source volume to fsaverage using dipy
+==================================================================
+
+This example demonstrates how to morph an individual subject source space to a
+common reference space. For this purpose dipy (http://nipy.org/dipy/) will be
+used in order to perform the necessary transforms to a fsaverage serving as
+reference space. The example uses parts of the MNE example lcmv beamformer
+pipeline (examples/inverse/plot_lcmv_beamformer_volume.py).
+The respective result will be morphed based on an affine transformation and a
+nonlinear morph, estimated based on respective transformation from the
+subject's anatomical T1 (brain) to fsaverage T1 (brain). Afterwards the
+transformation will be applied to the beamformer result. Affine transformations
+are computed based on the mutual information and nonlinear transformations
+will be performed as Symmetric Diffeomorphic Registration using the
+cross-correlation metric [1]_
+
+.. warning:: Please do not copy the patterns presented here for your own
+             analysis, this is example is purely illustrative.
+
+.. note:: This example does quite a bit of processing, so even on a
+          fast machine it can take a couple of minutes to complete.
+
+References
+----------
+.. [1] Avants, B. B., Epstein, C. L., Grossman, M., & Gee, J. C. (2009).
+        Symmetric Diffeomorphic Image Registration with Cross- Correlation:
+        Evaluating Automated Labeling of Elderly and Neurodegenerative Brain,
+        12(1), 26-41.
+
+"""
+# Author: Tommy Clausner <tommy.clausner@gmail.com>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+import mne
+from mne.datasets import sample
+from mne.beamformer import make_lcmv, apply_lcmv
+
+from dipy.align.imwarp import SymmetricDiffeomorphicRegistration
+from dipy.align.metrics import CCMetric
+from dipy.align.transforms import (TranslationTransform3D,
+                                   RigidTransform3D,
+                                   AffineTransform3D)
+from dipy.align.imaffine import (transform_centers_of_mass,
+                                 MutualInformationMetric,
+                                 AffineRegistration)
+import nibabel as nib
+
+# uncomment the line(s) below if you encounter problems with ssl certification
+# import ssl
+# if hasattr(ssl, '_create_unverified_context'):
+#     ssl._create_default_https_context = ssl._create_unverified_context
```

Comment:
Should not need this...?

Reply:
well the thing is, in theory it shouldn't be necessary, however I was not able to retrieve the example data without ssl verification ... so in my case it was not possible otherwise... Could also just remove it if you like... (?)

Reply:
yes you shouldn't need it, this is a problem with your Python presumably. Are you on an old 2.7 env?

Reply:
actually it's 2.7.15